pkg/manager/diff.go

```diff
@@ -73,8 +79,10 @@ func RunDiffFuzzer(ctx context.Context, baseCfg, newCfg *mgrconfig.Config, debug
 		}
 		new.http = diffCtx.http
 	}
-	diffCtx.Loop(ctx)
-	return nil
+	eg.Go(func() error {
+		return diffCtx.Loop(ctx)
+	})
+	return eg.Wait()
```

Comment:
If we want real cancellation, time.Sleep(15 * time.Minute) in Loop should wait of context cancellation as well.

Reply:
Done